frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -125,6 +125,7 @@ func httpHandler(h handler) http.Handler {
 			return
 		}
 		replyJSONOK(w, res)
+		log.Println("handling request(%p) ended %s %s", r, r.Method, r.URL.Path)
```

Comment:
remove r.Method and r.URL.Path from the ended log message, these are redundant in that statement.